### PR TITLE
[TESTED]Completed the pwd command

### PR DESCRIPTION
**Pull request by GoKo-Son626**
Added the `pwd` command to the user space， to output the current working directory

### CODE REVIEW

File: user/pwd.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ */
+#include "user.h"
+
+#define MAXPATH                 128
+
+int main(int argc, char *argv[]) {
+        char cwd[MAXPATH];
+
+        if (getcwd(cwd, MAXPATH)) {
+                printf("%s", "pwd:getcwd failed\n");
```
Comment (TroyMitchell911) [inline user/pwd.c]: Just return -1 don't output anything

Comment (GoKo-Son626) [reply user/pwd.c]: I‘ve been fixed it #27

Comment (GoKo-Son626) [reply user/pwd.c]: I‘ve been fixed it eb97f8c71e85c0b1daccd31128259d1a39c65a2a

Comment (GoKo-Son626) [reply user/pwd.c]: Addressed some indentation issues 282cc742e2ce4e7261a587c74cffbe4d95d43504

**Comment by TroyMitchell911** [comment]
Hey, Never modify two or more unrelated codes in one PR.

**Pull request merged**